### PR TITLE
Apply filter 

### LINKED ISSUE #66: Apply filter to the search page

### PR DESCRIPTION
**Pull request by Danik1702**
- Apply filter to result page
- Change some styles on result page

### CODE REVIEW

**Review by okhabr — APPROVED** [review]

**Review by Pashkaua — APPROVED** [review]
Good !

File: src/pages/result/utils/filter.util.ts [hunk 1]
```diff
@@ -4,9 +4,36 @@ import { PetProfile } from '../../../shared/interfaces';
...
+      high = 4;
+
+    if (typeof userInputValue === 'boolean') {
+      return petValue >= average;
+    } else if (typeof userInputValue === 'string') {
+      switch (userInputValue) {
+        case 'low':
+          return petValue <= low;
+        case 'averege':
+          return petValue === average;
```
Comment (ivan-bonk) [inline src/pages/result/utils/filter.util.ts:19]: Is this correct?
averege != average

**Review by ivan-bonk — APPROVED** [review]
Looks nice

File: src/pages/result/utils/filter.util.ts (hunk 1 above)
Comment (Danik1702) [reply src/pages/result/utils/filter.util.ts:19]: Yep, this is correct. 
And average is a variable which contains a number '3';
I know that I have here a spelling mistake, 'averege' is not correct, but this value I took from Pasha so I'll change it when Pasha changes it on his page. ))

**Pull request merged**